internal/conf/configuration.go

```diff
@@ -627,10 +637,10 @@ func (e *ExtensibilityPointConfiguration) ValidateExtensibilityPoint() error {
 		return validatePostgresPath(u)
 	case "http":
 		hostname := u.Hostname()
-		if hostname == "localhost" || hostname == "127.0.0.1" || hostname == "::1" || hostname == "host.docker.internal" {
+		if isStringInSlice(hostname, allowedHTTPHostNames) {
```

Comment:
should `allowedHTTPHostNames` default to the previously allowed hostnames (i.e. localhost, 127.0.0.1, etc)?

Reply:
Possible, initial consideration was that it might be better to set in CLI since we also set  "edge_runtime" which is not specific to Supabase.

There's also the option to set `localhost`, `127.0.0.1`, `hostname`.. etc by default and insert `edge_runtime` in the CLI. Am open to that.